run/deployment-previews/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
```

Comment:
Add a comment on what this includes. Also can you make this more consistent with other samples (https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/helloworld/Dockerfile)